src/main/resources/application.yaml

```diff
@@ -32,3 +37,12 @@ info:
     name: Learning Managements System
     description: Application provides functionality for creation of courses and students' further enrollment
     version: @project.version@
+
+logging:
+  level:
+    org:
+      hibernate:
+        SQL: debug
+      type:
+        descriptor:
+          sql: trace
```

Comment:
logs shouldn't be exposed from default configuration class, in case you will forget to disable them for cloud profile for example, you can expose some sensitive data in logs on production environment